src/workloads/docs/RandomDataFromDataset.yml

```diff
@@ -0,0 +1,21 @@
+SchemaVersion: 2018-07-01
+Owner: "@10gen/product-perf"
+Description: |
+  Demonstrate the ChooseFromDataset generator. The ChooseFromDataset generator takes one single
+  argument, a path to a file. It then returns random lines from that file. The file should exist
+  and shouldn't be empty. As an example you can use familynames.txt, names.txt and airports_codes.txt.
+  If you specify a relative path the pathfile will depend on your current working directory (cwd).
+  If running in evergreen, the relative path needs to be: src/genny/src/workloads/datasets/.
+
+Actors:
+- Name: Insert
+  Type: Insert
+  Threads: 1
+  Database: "test2"
+  Phases:
+  - Collection: "inserts"
+    Document:
```

Comment:
Could you increase the threads and add a use of the form 
`{^Join: {array: [{^ChooseFromDataset: {"path": "src/genny/src/workloads/datasets/names.txt"}}, " ", {^ChooseFromDataset: {"path": "src/genny/src/workloads/datasets/familynames.txt"}}]}}`
In addition to these examples, to show off the idea that these datasets are loaded only once into memory?

Reply:
I have added the join with 8 threads.